docs/Platforms.md

```diff
@@ -0,0 +1,14 @@
+
+## CIS Kubernetes Benchmark support
+
+kube-bench supports the tests for Kubernetes as defined in the [CIS Kubernetes Benchmarks](https://www.cisecurity.org/benchmark/kubernetes/).
```

Comment:
I think this isn't true since we have some tests that are not from CIS? and also the phrasing implies we support all the tests which we are not.